### ISSUE TITLE
Add Jellyfin Media Player to Microsoft Store

### ISSUE THREAD

**Issue by Cubevoid** [jellyfin/jellyfin-media-player#172]
**Is your feature request related to a problem? Please describe.**
Having the desktop app in the new Microsoft Store would make auto-updates easy on Windows. It would also improve discovery by new users and give them the confidence of installing from a trusted source, instead of from the GitHub repo.

**Describe the solution you'd like**
Having Jellyfin Media Player available in the new Microsoft Store (for Windows 10 and 11).

**Describe alternatives you've considered**
Auto-updates outside of the official store, but this does not solve the discoverability issue. On Linux for example, installing the flatpak version is easy and provides a convenient update mechanism.

**Additional context**
The new Microsoft store allows non-UWP apps, such as Firefox and Discord. So there should not be any technical barriers for adding Jellyfin Media Player there too.

**Comment by zjeffer** [comment]
Apparently there's a Github Action to do this: https://github.com/marketplace/actions/microsoft-store-publish

Seems pretty simple but a maintainer would have to do it.

**Comment by Fbonazzi** [comment]
This would be neat to have

**Comment by vslee** [comment]
Apparently it's on there already for XBox: https://www.microsoft.com/en-us/p/jellyfin/9p2drtg62qf8

**Comment by thornbill** [comment]
That's a different app.

**Comment by vslee** [comment]
Yes, but I think it means that the Jellyfin org already has partner status or whatever that is required to publish apps to the store.

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
